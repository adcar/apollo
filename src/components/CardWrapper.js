import React from 'react'
import { withStyles } from 'material-ui/styles'

const styles = theme => ({
	cardWrapper: {
		display: 'grid',
		gridTemplateColumns: 'repeat(auto-fit, minmax(320px, 1fr))',
		justifyItems: 'center',
		alignItems: 'center'
	}
})

const CardWrapper = ({ classes, children }) => (
	<div className={classes.cardWrapper}>{children}</div>
)

export default withStyles(styles)(CardWrapper)
