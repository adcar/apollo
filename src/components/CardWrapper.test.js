import React from 'react'
import CardWrapper from './CardWrapper'
import { shallow, mount, render } from 'enzyme'
import toJson from 'enzyme-to-json'

test('CardWrapper renders correctly', () => {
	const wrapper = shallow(<CardWrapper>Test Page!</CardWrapper>)
	expect(toJson(wrapper)).toMatchSnapshot()
})
