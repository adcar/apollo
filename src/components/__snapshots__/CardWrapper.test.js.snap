// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardWrapper renders correctly 1`] = `
<CardWrapper
  classes={
    Object {
      "cardWrapper": "CardWrapper-cardWrapper-1",
    }
  }
>
  Test Page!
</CardWrapper>
`;
